app/src/main/java/org/cru/godtools/ui/profile/MyProfileActivity.kt

```diff
@@ -25,15 +27,14 @@ class MyProfileActivity : BasePlatformActivity() {
     // region lifeCycle Calls
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        binding = ActivityMyProfileBinding.inflate(layoutInflater).also {
-            setContentView(it.root.id)
-        }
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_my_profile)
```

Comment:
`DataBindingUtil.setContentView` can cause crashes. we already had to rework logic in mpdx to deal with crashes from this method.

Reply:
https://github.com/CruGlobal/mpdx-android/pull/891